vllm/sequence.py

```diff
@@ -706,10 +706,35 @@ def prompt_adapter_num_virtual_tokens(self) -> int:
         return self.prompt_adapter_request.prompt_adapter_num_virtual_tokens\
                          if self.prompt_adapter_request else 0
 
-    def init_multi_step(self, num_scheduler_steps: int) -> None:
-        self.state.num_steps = num_scheduler_steps
+    def init_multi_step(self, num_steps: int) -> None:
```

Comment:
I'll prefer to keep the argument name `num_scheduler_step` to be more clear.

Reply:
I wanted to differentiate this between the arguments of `init_multi_step_from_lookahead_slots`. also this way it just acts as a setter and is semantically closer to `self.state.num_steps`.  
I believe calling it `num_steps` serves as a caution when the caller is setting it directly and not via `init_multi_step_from_lookahead_slots`.

Also, in the future, this need not be equal to the `num_scheduler_steps` always. it can be smaller in case we want to lesser steps dynamically. We can also change it back to `num_steps` when that happens.

Happy to change it if you still feel strongly about it.

Reply:
If `init_multi_step` is no longer called outside SequenceGroup, can we change the name to `_init_multi_step` for clarity?